File: src/backend/BusinessPartners.Api/src/main/java/com/murilobeltrame/businesspartnersapi/models/repositories/BusinessPartnersRepository.java
package com.murilobeltrame.businesspartnersapi.models.repositories;

import com.murilobeltrame.businesspartnersapi.models.entities.BusinessPartner;
import org.springframework.data.repository.CrudRepository;

public interface BusinessPartnersRepository extends CrudRepository<BusinessPartner, Long> { }

File: src/backend/BusinessPartners.Api/Models/Repositories/BusinessPartnersRepository.cs
using BusinessPartners.Api.Models.Entities;
using SharedKernel.Models;

namespace BusinessPartners.Api.Models.Repositories
{
    public class BusinessPartnersRepository : BaseRepository<BusinessPartner>
    {
        public BusinessPartnersRepository(ApplicationContext context) : base(context) { }
    }
}

